code/frontend/src/app/features/settings/account/two-factor-card.component.html

```
@@ -61,10 +61,13 @@
           [value]="twoFaCode()"
           (valueChange)="twoFaCode.set($event)"
         />
+        @if (retryCountdown() > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)